sklearn/linear_model/ridge.py

```diff
@@ -643,6 +643,7 @@ class Ridge(_BaseRidge, RegressorMixin):
           normalize=False, random_state=None, solver='auto', tol=0.001)
 
     """
+
```

Comment:
Please avoid introducing unnecessary and unrelated changes like this. It makes it hard to review your work, and may introduce merge conflicts for other changes in the works.